.github/workflows/run-tests.yml

```diff
@@ -13,34 +13,30 @@ jobs:
       fail-fast: false
       matrix:
         php:
-          - 7.4
-          - 7.3
+          - 8.3
+          - 8.2
+          - 8.1
         laravel:
-          - 8.*
-          - 7.*
+          - 11.*
+          - 10.*
         deps:
           - lowest
           - latest
         include:
+          - {php: '8.3', laravel: '11.*'}
+          - {php: '8.2', laravel: '11.*'}
+
+          - {php: '8.3', laravel: '10.*'}
           - {php: '8.2', laravel: '10.*'}
           - {php: '8.1', laravel: '10.*'}
-
-          - {php: '8.2', laravel: '9.*'}
```

Comment:
What was the reason for ditching Laravel v9?

Laravel v11 was just released a few days ago.
Laravel v9 just stopped receiving security fixes last month.

We are going to do a major version bump, so I think it's fine to say "we support versions that are supported by the community", but I'm wondering if there were breaking changes between the v9 and v10-11.

Reply:
I dropped 9 because of dependency problems between PHP 8.0, testbench, and phpunit, and I didn't want to spend the time to have conditional code in the tests to handle the older version of phpunit. Since Laravel 9 users can still use the 3.x version of the library, and since 9 is on the way out anyway, I think it's ok to drop it in a new major version of this library.